heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -113,7 +108,7 @@ TMaster::TMaster(const std::string& _zk_hostport, const std::string& _topology_n
   tmaster_location_->set_master_port(master_port_);
   tmaster_location_->set_stats_port(stats_port_);
   DCHECK(tmaster_location_->IsInitialized());
-  EstablishTMaster(EventLoop::TIMEOUT_EVENT);
+  EstablishPackingPlan(EventLoop::TIMEOUT_EVENT);
```

Comment:
Why is the proposed approach a better one? With the proposed approach tmaster would register itself as master, which would result in stream managers trying to connect to it. They will fail until tmaster continues it's initialization and opens its server socket, which seems undesirable.